content/docs/quickstart.mdx

```diff
@@ -22,74 +22,216 @@ keywords:
   ]
 ---
 
-import ConfigDocker from '@site/content/examples/config/config.docker.yaml.md';
-import DockerCompose from '@site/content/examples/docker/basic.docker-compose.yml.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Pomerium Zero Quickstart
+
+The Zero Quickstart shows you how to install and run Pomerium Zero in a Docker container.
+
+## Before you start
+
+- Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- Create a (free) [**Pomerium Zero account**](https://console.pomerium.app/create-account)
+
+## Get Pomerium Zero configuration
+
+In the Zero onboarding screen, select the **Docker** tab and copy the Docker Compose configuration. (If you selected **Finish** before copying the Docker configuration, we've provided a copy below.)
+
+Paste the configuration in a `compose.yaml` file:
+
+```yaml title="compose.yaml" {8,15} showLineNumbers
+services:
+  pomerium:
+    image: pomerium/pomerium:v0.27.0
+    ports:
+      - 443:443
+    restart: always
+    environment:
+      POMERIUM_ZERO_TOKEN: <CLUSTER_TOKEN>
+      XDG_CACHE_HOME: /var/cache
+    volumes:
+      - pomerium-cache:/var/cache
+    networks:
+      main:
+        aliases:
+        - verify.<CLUSTER_STARTER_SUBDOMAIN>.pomerium.app
+  verify:
+    image: cr.pomerium.com/pomerium/verify:latest
+    networks:
+      main:
+        aliases:
+        - verify
+
+networks:
+  main: {}
+
+volumes:
+  pomerium-cache:
+```
 
-# Run Pomerium Core With Docker
+Replace the following values:
 
-Run Pomerium Core with Docker containers in **under 5 minutes**.
+- **Line 8**: Replace `CLUSTER_TOKEN` with the cluster token generated for you in the onboarding config
+- **Line 15**: Replace `CLUSTER_STARTER_SUBDOMAIN` with your generated starter subdomain
 
-The Core quickstart uses Pomerium's [**Hosted Authenticate Service**](/docs/capabilities/hosted-authenticate-service), but you can also configure a [**self-hosted authenticate service**](/docs/capabilities/self-hosted-authenticate-service) to integrate with Pomerium.
+:::info Where's my **Starter Domain?**
 
-## Prerequisites
+You can find your **Starter Domain** at the top of the Zero Console in the navigation bar.
 
-[Docker] and [Docker Compose]
+:::
 
-## Configure Pomerium
 
-Create a `config.yaml` file in the root of your project.
+## Connect to Pomerium Zero
 
-Add the configuration below to `config.yaml`:
+Next, deploy Pomerium Zero with the following command:
 
-<ConfigDocker />
+```bash
+docker compose up -d
+```
 
-## Configure Docker
+This command will deploy a local cluster running Pomerium in a Docker container. If your cluster deploys successfully, the Pomerium Zero cloud will attempt to secure a connection to your cluster.
```

Comment:
I think we could give some better explanation here. In particular "Pomerium Zero cloud will attempt to secure a connection to your cluster" doesn't make sense to me. Maybe we could say something about how this starts Pomerium in a mode where it connects to the cloud service?

We might also want to hold off on talking about a "cluster" just yet... this might be confusing to some folks if they assume "cluster" implies multiple replicas.

Reply:
If we want to start being more consistent with terminology, this might be a good time to mention "Zero-managed mode", and that that really means Pomerium connects to the Pomerium Zero cloud?

I see your point with the mention of a cluster. I'm in favor of removing it, we just need to agree on how we want to replace it.